eclipse-bluechi/workspace/workloads/ibeji.kube

```diff
@@ -16,5 +16,8 @@
 Yaml=ibeji.yml
 
 # Commented to disable the service to automatically start
-# [Install]
-# WantedBy=default.target
+[Install]
+WantedBy=default.target
+
+[Unit]
+Wants=chariott.service
```

Comment:
If more than one service can be listed here, I would suggest adding agemo.

Ibeji depends on Chariott and Agemo